joomla/joomla-cms
# [5.2] TinyMCE 7

Pull Request for Issue # .

### Summary of Changes
The majority of changes with 7.x do not impact us as they are for the premium version and the premium plugins. Other changes are really just bug fixes and accessibility improvements.

The full changelog for the open source release of tinymce can be found [here](https://www.tiny.cloud/docs/tinymce/latest/changelog/)

Set tinymce to be used under the gpl licence. https://www.tiny.cloud/docs/tinymce/latest/license-key/#use-tinymce-with-the-gplv2-license - without this change a console warning is displayed

The one change that does need testing is sandbox_iframes

### Testing Instructions
1. Test all editor-xtd buttons 

2. Test the source code editor 
![image](https://github.com/user-attachments/assets/5d0bcba4-d0b5-405d-b663-4b94033e0576)

3. Test any third party plugin eg https://github.com/brianteeman/slashtools
This uses one of the few breaking changes so should fail https://www.tiny.cloud/docs/tinymce/latest/migration-from-6x/#the-autocompleter-ch-configuration-property-has-been-removed-use-the-trigger-property-instead I will make a new release of it but I am not aware of any other plugin using this.

4. Test drag and drop images

5. Test content templates
![image](https://github.com/user-attachments/assets/83e64307-424a-4b32-ab0f-a584f7becedc)

### Link to documentations
Please select:
